tests/entrypoints/openai/basic/test_chat_echo.py

```diff
@@ -35,24 +15,20 @@ async def client(server):
 
 
 class TestCase(NamedTuple):
-    model_name: str
     echo: bool
 
 
 @pytest.mark.asyncio
 @pytest.mark.parametrize(
     "test_case",
-    [
-        TestCase(model_name=MODEL_NAME, echo=True),
-        TestCase(model_name=MODEL_NAME, echo=False)
-    ],
+    [TestCase(echo=True), TestCase(echo=False)],
```

Comment:
You can probably delete `TestCase` completely as it doesn't really do anything (all it does now is wrap a single bool)